### PR TITLE
[4.0] Fix unable to save user options

### PR DESCRIPTION
**Pull request by Quy**
This reverts commit dac68625e972b29396bbeba665b2299ab605a092.

Pull Request for Issue #27667. Regression from #27556. 

### Summary of Changes
Fix unable to save user options.

### Testing Instructions
Go to Configuration > Users
Change any setting and save

### CODE REVIEW

File: libraries/src/Form/FormField.php [hunk 1]
```diff
@@ -1129,7 +1129,7 @@ public function validate($value, $group = null, \Joomla\Registry\Registry $input
 		{
 			$subForm = $this->loadSubForm();
 
-			if ($this->multiple && $value)
+			if ($this->multiple)
 			{
 				foreach ($value as $key => $val)
```
Comment (SharkyKZ) [inline libraries/src/Form/FormField.php]: I think `$value` can be cast as array here. Or `$value` check can be added before `foreach`. This should keep the notice fixed.

**Comment by jwaisner** [comment]
Getting `Invalid controller class: display` after applying PR and cannot access Global Config.

**Comment by brianteeman** [comment]
@jwaisner delete the file libraries\autoload_psr4.php - it will be regenerated and should fix that

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on a3106482b8bf219c6cfbb9becc926e8dcf3a68d6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27671">issues.joomla.org/tracker/joomla-cms/27671</a>.</sub>

File: libraries/src/Form/FormField.php (hunk 1 above)
Comment (Quy) [reply libraries/src/Form/FormField.php]: Please provide code you have in mind. Thanks.

Comment (SharkyKZ) [reply libraries/src/Form/FormField.php]:
```
if ($this->multiple)
{
	if ($value)
	{
		foreach ($value as $key => $val)
...
```

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on eeab4336b8ac0f49ead5a5100f9159f73e32d169<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27671">issues.joomla.org/tracker/joomla-cms/27671</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on eeab4336b8ac0f49ead5a5100f9159f73e32d169<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27671">issues.joomla.org/tracker/joomla-cms/27671</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27671">issues.joomla.org/tracker/joomla-cms/27671</a>.</sub>

**Pull request merged**